web/frontend/src/components/slam.vue

```diff
@@ -48,6 +51,23 @@ const loaded2d = $computed(() => (pointcloud !== undefined && pose !== undefined
 let slam2dTimeoutId = -1;
 let slam3dTimeoutId = -1;
 
+const moveClicked = $computed(() => {
+  for (const element of props.operations) {
+    if (element.op.method.includes('MoveOnMap')) {
+      return true;
+    }
+  }
+  return false;
+});
+
+// get all resources which are bases
+const baseResources = $computed(() => filterResources(props.resources, 'rdk', 'component', 'base'));
+
+// allowMove is only true if we have a base, there exists a destination and there is no in-flight MoveOnMap req
+const allowMove = $computed(() => (
+  (baseResources !== undefined) &&
+  (baseResources.length > 0) && updatedDest && !moveClicked));
```

Comment:
I thought we said we would only show the button if there was a single base? Otherwise, which base is used would be arbitrary.